CodeSign.targets

```diff
@@ -1,97 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why move this into build.proj?

Reply:
Because it was in build.proj and got moved out. But, it wasn't changed almost at all. So, this file relied on properties being defined in build.proj. And, build.proj relied on properties/targets being defined in this. It just didn't make any sense to have a separate if they can't be independent of each other. Also, I simplified the usage in build.proj for these targets.